code/modules/mob/living/carbon/species.dm

```diff
@@ -410,6 +410,38 @@
 
 	H.stop_tail_wagging(1)
 
+/datum/species/vulpkanin
+	name = "Vulpkanin"
+	icobase = 'icons/mob/human_races/r_vulpkanin.dmi'
+	deform = 'icons/mob/human_races/r_vulpkanin.dmi'
+	path = /mob/living/carbon/human/vulpkanin
+	default_language = "Galactic Common"
+	language = "Canilunzt"
+	primitive = /mob/living/carbon/monkey/vulpkanin
+	tail = "vulptail"
+	unarmed_type = /datum/unarmed_attack/claws
+	darksight = 8
+	slowdown = -1
+	brute_mod = 1.1
+	burn_mod = 1.1
```

Comment:
these need to be 1.5, or maybe even 2

1.1 just isn't enough for a race that when stacking meth, morphine, and ephedrine can get -4 movespeed